curation-migration-datasync/config.ts

```diff
@@ -32,9 +32,9 @@ const config = {
     },
   },
   db: {
-    secretId: process.env.DATABASE_SECRET_ID || '',
+    readSecretId: process.env.READ_DATABASE_SECRET_ID || '',
+    writeSecretId: process.env.WRITE_DATABASE_SECRET_ID || '',
```

Comment:
Curious as to why we want them as ` '' ` and not let them be `undefined`? 

I see we are using them above, would that cause things to throw when instantiating the `lambdaConfig`?

Reply:
These are always set in deployed environments. We could default to something else, but an empty string is sufficient for our tests